{{cookiecutter.github_project_name}}/.github/workflows/tests.yml

```
@@ -0,0 +1,40 @@
+name: Python and JS tests
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.10]
```

Looks like we might need to do something similar to https://github.com/jupyterlab-contrib/jupyterlab-app-cookiecutter/pull/21 here too?

```suggestion
        python: ["3.10"]
```

---

Thanks!